docs/calamari/SDK.md

```diff
@@ -0,0 +1,292 @@
+# 🛠 Calamari SDK
+
+This package implements a Javascript SDK for connecting with the Manta Network. API end points to connect to Calamari:
+
+## Full Nodes
+
+- provided by Manta: `wss://ws.calamari.systems`
+- provided by third party: `wss://calamari.api.onfinality.io`
+
+## Asset API (Javascript)
+
+- KMA: `api.query.system.account(AccountId)`
```

Comment:
Maybe we should tell readers what 8, 9 means.
And list all current asset ids we're supporting.
it's unnecessary to write several line of this code.
We can add just a line of code like this
```
api.query.assets.account(asset_id, AccountId)
```

Reply:
Good point.